pkg/streaming/commands/processor_create.go

```diff
@@ -167,26 +165,23 @@ func NewProcessorCreateCommand(ctx context.Context, c *cli.Config) *cobra.Comman
 	return cmd
 }
 
-// Parse parameter-stream name bindings, returns parameter and stream names
+// Parse stream bindings, returns potential aliases and stream names.
 //
 // Valid values are:
-//  - ${PARAM_NAME}:${STREAM_NAME}
+//  - ${ALIAS}:${STREAM_NAME}
```

Comment:
let's reverse stream name and alias so that shell completion is more meaningful. The stream name can complete, the alias cannot

Reply:
I don't think completion itself is a good excuse to reverse. The alias is the "key" in the "key:value" concept. It should be unique, and we bind _that stream_ *to* _that alias_. We can adapt completion to complete stream names when doing `alias:str<TAB><TAB>`